spec/components/previews/marketplace/product_component_preview.rb

```diff
@@ -0,0 +1,9 @@
+class Marketplace
+  class ProductComponentPreview < ViewComponent::Preview
+    # @param name
+    # @param price
+    def card(name: "Fancy Pants", price: 8.5)
+      render(ProductComponent.new(product: Product.new(name: name, price: price)))
+    end
+  end
+end
```

Comment:
Does a spec/components/previews/marketplace/deliveryarea_component_preview.rb and a taxrate_component_preview.rb need to be created too in a different PR?

Reply:
`TaxRateComponentPreview` already exists, so you should be able to go to https://localhost:3000/components/ and see it under the `Marketplace` heading.

I'd love to see a `DeliveryAreaComponentPreview`!!!

Reply:
I'll work on that next.